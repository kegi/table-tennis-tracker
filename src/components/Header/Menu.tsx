import React, { Component } from 'react'
import { NavLink } from 'react-router-dom'

export default class Menu extends Component {
  render () {
    return (
      <nav id="header-menu">

        <ul>
          <li>
            <NavLink to="/" exact={true} activeClassName="active">
              homepage
            </NavLink>
          </li>

          <li>
            <NavLink to="/page-1" exact={true} activeClassName="active">
              static page
            </NavLink>
          </li>

          <li>
            <NavLink to="/redux" exact={true} activeClassName="active">
              Redux example
            </NavLink>
          </li>

          <li>
            <NavLink
              to="/i-do-not-exists"
              exact={true}
              activeClassName="active"
            >
              missing page
            </NavLink>
          </li>

          <li>
            <NavLink
              to="/error"
              exact={true}
              activeClassName="active"
              style={{ color: '#760000' }}
            >
              Test error
            </NavLink>
          </li>

        </ul>

      </nav>
    )
  }
}
