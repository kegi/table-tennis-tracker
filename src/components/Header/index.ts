export { default } from './Header'
export { default as HeaderMenu } from './Menu'
