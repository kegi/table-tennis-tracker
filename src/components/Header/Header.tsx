import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import Menu from './Menu'

import './scss/header.scss'

import logo from './../../../assets/images/logo.png'

export default class Header extends Component {
  render () {
    return (
      <header id="header">

        <div className="col-3">
          <Link to="/">
            <img src={logo} alt="Logo" className="logo"/>
          </Link>
        </div>
        <div className="col-9">

          <Menu />

        </div>

      </header>
    )
  }
}
