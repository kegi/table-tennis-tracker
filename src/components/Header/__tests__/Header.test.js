import React from 'react'
import Header from './../'
import { BrowserRouter } from 'react-router-dom'
import { mount } from 'enzyme'

describe('component:Header', () => {
  it('render correctly', () => {
    const component = mount(
      <BrowserRouter>
        <Header/>
      </BrowserRouter>
    )
    expect(component).toMatchSnapshot()
  })
})
