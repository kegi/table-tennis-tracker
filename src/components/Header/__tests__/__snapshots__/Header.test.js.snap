// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component:Header render correctly 1`] = `ReactWrapper {}`;
